src/preferences/dialog/dlgprefeq.cpp

```diff
@@ -45,6 +45,9 @@ DlgPrefEQ::DlgPrefEQ(QWidget* pParent, EffectsManager* pEffectsManager, UserSett
     m_pOutputEffectRack = m_pEffectsManager->getOutputsEffectRack();
 
     setupUi(this);
+
+    loadSettings();
```

Comment:
Leave a comment why the settings must be loaded here and not later?

Reply:
Not really. It's fixed in main already.
Also the comment would actually belong to/into `slotNumDecksChanged` which is doing the reset.